example_config_files/amplicon_studies.csv

```diff
@@ -0,0 +1,5 @@
+study,studyPath
```

Comment:
i dont mind having the example files here, but wonder if there is a reason? 

it also seems worth this examples dir having its own readme file, pointing people to the github repos where they can see these files as they are in production